src/tests/hyeth.test.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  getMainnetTestFactory,
+  QuoteTokens,
+  SignerAccount4,
+  TestFactory,
+  transferFromWhale,
+  wei,
+} from './utils'
+
+const { eth, hyeth, usdc } = QuoteTokens
+
+describe('hyETH', () => {
+  const indexToken = hyeth
+  let factory: TestFactory
+  beforeEach(async () => {
+    const signer = SignerAccount4
+    factory = getMainnetTestFactory(signer)
+  })
+
+  test('can mint with ETH', async () => {
+    await factory.fetchQuote({
+      isMinting: true,
+      inputToken: eth,
+      outputToken: indexToken,
+      indexTokenAmount: wei('1'),
```

Comment:
can we test with much larger amounts? like `1000 ether` ? that will let us know if any of the integrations quotes break with large values

Reply:
Highest I could do for now is 300 but that's 1+m. Should be enough?